addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -1,18 +1,20 @@
 import csv
 
+from freezegun import freeze_time
 from odoo import Command
-from odoo.exceptions import ValidationError
+from odoo.exceptions import UserError, ValidationError
 from odoo.tests import tagged
-from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.addons.account.tests.common import AccountTestInvoicingHttpCommon
 from odoo.addons.l10n_id_efaktur.models.account_move import FK_HEAD_LIST, LT_HEAD_LIST, OF_HEAD_LIST, _csv_row
 from odoo.exceptions import RedirectWarning
+from markupsafe import Markup
 
 
 @tagged('post_install', '-at_install', 'post_install_l10n')
-class TestIndonesianEfaktur(AccountTestInvoicingCommon):
+class TestIndonesianEfaktur(AccountTestInvoicingHttpCommon):
 
     @classmethod
-    @AccountTestInvoicingCommon.setup_country('id')
+    @AccountTestInvoicingHttpCommon.setup_country('id')
```

Comment:
I see why you changed (download tests) but I think it's better to split in two files; one for the download test and one with the other tests.